data/data/coreos/OWNERS

```diff
@@ -0,0 +1,20 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# This file should mirror the list of users defined on coreos-approvers in:
+# https://github.com/openshift/release/blob/master/OWNERS_ALIASES
+
+approvers:
+  - miabbott
```

Comment:
Please add this list as an alias in OWNER_ALIAS instead of here. That way we can use the same list for multiple OWNER files.
https://github.com/openshift/installer/blob/c653122585431b6beb9759efc0a67eb8de4a6d28/OWNERS_ALIASES#L3